.gitignore

```diff
@@ -6,3 +6,4 @@ bower_components
 Gemfile.lock
 styleguide/css/*.css
 backstop_data/bitmaps_test
+npm-debug.log
```

Comment:
Bikeshedding...

My comment was a recommendation and I have to make this same recommendation all of the time to developers all over the world.